.travis.yml

```diff
@@ -10,7 +10,7 @@ install:
   - composer install --no-interaction --prefer-source
 
 script:
-  - ./vendor/bin/phpcs
+  - ./vendor/bin/phpcs -n
```

Comment:
Und das ist so schlau?

Reply:
Das ist notwendig, damit Warnungen den Build nicht abbrechen lassen. Sollte so auch in anderen Projekten umgesetzt werden.

Reply:
Ich würde es besser finden, wenn die Warnungen trotzdem ausgegeben werden.

Reply:
Ich definitiv auch, da bin ich ganz bei dir!
Aber das Flag zu entfernen sorgt, wie im vorherigen Kommentar beschrieben, dafür das der Build bei Warnungen fehlschlägt.

Reply:
https://stackoverflow.com/questions/16039362/how-can-i-suppress-phpcs-warnings-using-comments

Reply:
Gefällt dir diese Lösung besser?

Reply:
Wie lange soll der PR hier noch offen bleiben ... ?

Lasst phpcs doch zweimal laufen. Einmal ohne und einmal mit Warnungen. Der mit Warnungen könnt ihr bei Travis als "darf fehlschlagen" (allow_failures) markieren. So mach ich dies zum Beispiel auch im Flagception Bundle:
https://github.com/bestit/flagception-bundle/blob/master/.travis.yml

@nhardeweg @bestit-lubisch @bestit-tkellner